config/environments/production.rb

```diff
@@ -35,7 +35,7 @@
   # config.assume_ssl = true
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = false
```

Comment:
Just as some background information for this;

- when we're doing container healthchecks, they're happening over [HTTP](https://github.com/trade-tariff/trade-tariff-platform-aws-terraform/blob/ab616fb3d45cbaa2dcdccb76c80b31da17cc93d9/environments/common/application-load-balancer/alb.tf#L34) currently.
- `force_ssl` responds to those healthcheck requests with 301s pointing to HTTPS.
- the healthcheck receives 301 instead of 200 and marks the app as unhealthy.
- the app never starts fully.

So we can change our load balancer healthchecks to HTTPS and then set `force_ssl = true` in all the Rails apps. I think that's a better way forward here. I'll prep a quick PR on the infrastructure repo to do this.